lib/Sema/DerivedConformanceDifferentiable.cpp

```diff
@@ -281,6 +299,139 @@ deriveBodyDifferentiable_move(AbstractFunctionDecl *funcDecl, void *) {
   return deriveBodyDifferentiable_method(funcDecl, C.Id_move, C.Id_along);
 }
 
+/// Synthesize body for a `Differentiable` method requirement.
+static std::pair<BraceStmt *, bool>
+deriveBodyDifferentiable_zeroTangentVectorInitializer_getter(
```

Comment:
It would be useful to have comments inline with pseudocode corresponding to the code that is being generated.

There is some pseudocode but it's confusingly unrelated to the actual code that is being generated. Like the "`x.move(along: direction.x)`". I don't think it's synthesizing any `move(along:)` calls here.

Reply:
Thanks for the catch! I copied the code for synthesized `move(along:)` and forgot to update comments.

Comments are fixed now.